src/components/HorizontalCard.tsx

```diff
@@ -33,20 +13,18 @@ export interface IProps {
 
 const HorizontalCard: React.FC<IProps> = ({
   emoji,
-  title,
-  description,
   children,
   className,
   emojiSize,
 }) => (
-  <StyledCard className={className}>
-    <Emoji size={emojiSize} text={emoji} />
-    <Content>
-      <Title>{title}</Title>
-      <Description>{description}</Description>
+  <Flex borderRadius="base" className={className}>
+    <Emoji fontSize={emojiSize} text={emoji} />
```

Comment:
We were passing a number to `emojiSize` which represented `rem` values. Now we are going to use `fontSize` which is not expecting a `rem` value by default.

We have 2 options here.
- We modify this to be something like ```fontSize={`${emojiSize}rem`}```
- Or (I prefer this one) we change the name of the prop to `emojiFontSize` & we update the prop in all the components using this component.
```tsx
// e.g. in `src/pages/stablecoins.tsx`
// instead of passing
emojiSize={3}

// it would be
emojiFontSize="5xl"
// since 3rem=5xl in Chakra. Ref.: https://chakra-ui.com/docs/styled-system/theme#typography
```